packages/truffle-contract-schema/test/networks.js

```diff
@@ -0,0 +1,54 @@
+var contract = require("truffle-contract");
+
+var Schema = require("../index.js");
+var assert = require("assert");
+
+var MetaCoin = require("./MetaCoin.json");
+
+describe("networks", function() {
+  var MetaCoinContract;
+
+  before(() => {
+    MetaCoinContract = contract(MetaCoin);
+    MetaCoinContract.setNetwork(9999);
+    MetaCoinContract.address = "0x1111111111111111111111111111111111111111";
+    MetaCoinContract.transactionHash =
+      "0x1111111111111111111111111111111111111111111111111111111111111111";
+  });
+
+  it.only("normalized has correct events", function() {
```

Comment:
Oh shoot, gotta get rid of the `.only`, or otherwise none of the other tests will run!

Reply:
```suggestion
  it("normalized has correct events", function() {
```